.github/workflows/pre-commit-auto-fix.yml

```
@@ -0,0 +1,78 @@
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+      pull-requests: write
+    if: github.event_name == 'workflow_dispatch' ||
+        (github.event_name == 'issue_comment' &&
+         contains(github.event.comment.body, '/fix')) ||
+        github.event_name == 'pull_request'
+    steps:
+      - name: Get PR branch
+        id: pr-info
+        run: |
```

actionlint (1.7.11)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)